database/dataGeneration.js

```diff
@@ -0,0 +1,30 @@
+const faker = require("faker");
+const fs = require("fs");
+const csvWriter = require("csv-write-stream");
+var writer = csvWriter();
+
+const generateFilmography = () => {
+  let movies = [];
+  for (let i = 1; i < Math.floor(Math.random() * 6) + 1; i++) {
+    movies.push(faker.random.word());
+  }
+  return movies;
+};
+
+const generateActors = () => {
```

Comment:
Maybe consider using console.time and console.timeEnd to see how long the data generation takes?